CMakeLists.txt

```diff
@@ -651,14 +653,59 @@ list(APPEND BUN_RAW_SOURCES "${BUN_WORKDIR}/codegen/WebCoreJSBuiltins.cpp")
 # endif()
 
 # --- Runtime.js ---
-if(NOT NO_CODEGEN)
-    add_custom_command(
-        OUTPUT "src/fallback.out.js"
-        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
-        COMMAND "${ESBUILD}" "--target=esnext" "--bundle" "src/fallback.ts" "--format=iife" "--platform=browser" "--minify" "--outfile=src/fallback.out.js"
-        DEPENDS "src/fallback.ts"
-    )
-endif()
+#if(NOT NO_CODEGEN)
+#    add_custom_command(
+#        OUTPUT "src/fallback.out.js"
+#        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+#        COMMAND "${ESBUILD}" "--target=esnext" "--bundle" "src/fallback.ts" "--format=iife" "--platform=browser" "--minify" "--outfile=src/fallback.out.js"
+#        DEPENDS "src/fallback.ts"
+#    )
+#endif()
+
+# --- Runtime JS ---
+add_custom_command(
+    OUTPUT "${BUN_SRC}/runtime.out.js"
+    COMMAND "${ESBUILD}" "--define:process.env.NODE_ENV=\"${NODE_ENV}\"" "--target=esnext" "--bundle" "--format=iife" "--platform=browser" "--global-name=BUN_RUNTIME" "--minify" "--external:/bun:*" "${BUN_SRC}/runtime/index.ts" "--outfile=${BUN_SRC}/runtime.out.js"
```

Comment:
can you set node_env to production always for this. the worry is because they live in source you might get an incorrect build if you have debug and release builds.

it would make sense to take these out of src/, but ci currently expects the exact output paths and fixing all that might be a bit tricky.

Reply:
Ah all good currently it's set at the top of the file depending on the build. Debug builds development and release builds production, I'll change this to always production